src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -83,6 +84,11 @@ public void writeTo(Appendable out) throws IOException {
 
   /** Writes this to {@code directory} as UTF-8 using the standard directory structure. */
   public void writeTo(Path directory) throws IOException {
+    writeTo(directory, UTF_8);
+  }
+
+  /** Writes this to {@code directory} as UTF-8 using the standard directory structure. */
```

Comment:
Javadoc needs fixing as this is no longer utf8 only.

Reply:
Thank you for the hint! :-)